tests/scenarios/delegated-grant.spec.ts

```diff
@@ -464,7 +464,60 @@ export function testDelegatedGrantScenarios(): void {
     xit('should not allow entity using a non-delegated grant as a delegated grant to invoke delete', async () => {
     });
 
-    xit('should evaluate scoping correctly when invoking a delegated grant to write', async () => {
+    it('should fail if delegated grant has a mismatching protocol scope - write', async () => {
+      // scenario:
+      // 1. Alice creates a delegated grant for device X to act as her for a protocol that is NOT email protocol
+      // 2. Bob has email protocol configured for his DWN
+      // 3. Device X attempts to use the delegated grant to write an email to Bob as Alice
+      // 4. Bob's DWN should reject Device X's message
+      const alice = await DidKeyResolver.generate();
```

Comment:
Clarity:

Maybe adding a comment that the protocol allows anyone can write, or even showing the control test of Alice being able to write could be helpful.

Reply:
Updated the comment.